ims/einstein/ceph.py

```diff
@@ -48,7 +48,7 @@ def __init_context(self):
 
     @trace
     def __init_cluster(self):
-        cluster = rados.Rados(rados_id=self.rid, conffile=self.r_conf)
+        cluster = rados.Rados(conffile=self.r_conf)
```

Comment:
BMI wouldn't start up without this change strangely for me. So I had to make this change. Can you check if einstein and picasso start up without this change? 

FWIW, it also seems redundant since `rados_id` already exists in the conf.